### ISSUE TITLE
bodybuilder v2 ignore empty params

### ISSUE THREAD

**Issue by ferronrsmith** [danpaz/bodybuilder#87]
This bug is only present on the [prerelease-bodybuilder-2](https://github.com/danpaz/bodybuilder/tree/prerelease-bodybuilder-2) branch.

Bug:

Checking for a property to have an empty string (`""`) creates unwanted ES query.

```javascript
// prerelease-bodybuilder-2 branch
var body = new Bodybuilder()
  .filter('term', 'name', '')
  .body.build('v2')
```

gives
```javascript
{"query":{"bool":{"filter":{"term":{"field":"name"}}}}}
```

but using http://bodybuilder.js.org/ with 
```javascript
new Bodybuilder()
.filter('term', 'name', '')
        .build('v2')
```

produces:

```javascript
{
  "query": {
    "bool": {
      "filter": {
        "term": {
          "name": ""
        }
      }
    }
  }
}
```

**Comment by danpaz** [comment]
Good catch, thanks. seems like it's due to "" being falsey. Maybe this can be addressed in #86.

**Comment by danpaz** [comment]
This should be addressed in the latest merge to `prerelease-bodybuilder-2` branch, please let me know if it's not working for you.
